client/components/deposits-overview/index.tsx

```diff
@@ -98,10 +95,11 @@ const DepositsOverview: React.FC = () => {
 			</CardHeader>
 
 			{ /* Deposit schedule message */ }
-			{ isNextDepositExpected && !! account && (
+			{ isDepositsUnrestricted && !! account && (
 				<CardBody className="wcpay-deposits-overview__schedule__container">
 					<DepositSchedule
 						depositsSchedule={ account.deposits_schedule }
+						showNextDepositDate={ availableFunds !== 0 }
```

Comment:
Is this intentional to show next deposit date string if available balance is not 0? That means if it's negative too.

For example, every time I create an instant deposit, not sure why the available balance becomes negative even though the total balance (available + pending) is correct.

In this case (when available balance is negative), I can see the next deposit date string.

![Screenshot 2023-12-12 at 23 40 10](https://github.com/Automattic/woocommerce-payments/assets/73803630/34cc89e6-d420-4398-a90c-63c472be20d2)

Reply:
Or you meant

```suggestion
						showNextDepositDate={ availableFunds > 0 }
```

Reply:
I think you're right, we should go with showing this if there is a positive available balance.

Reply:
Resolved in b6b2523f798de979fdc446d86d2eda648a23d59d